examples/jsm/renderers/nodes/accessors/WVPNode.js

```diff
@@ -0,0 +1,32 @@
+import Node from '../core/Node.js';
+import CameraNode from '../accessors/CameraNode.js';
+import ModelNode from '../accessors/ModelNode.js';
+import OperatorNode from '../math/OperatorNode.js';
+import PositionNode from '../accessors/PositionNode.js';
+
+class WVPNode extends Node {
```

Comment:
Shouldn't it be named `MVPNode`? AFAIK, the academical correct term for this specific matrix is `Model View Projection matrix`.

Reply:
I think in `GLPositionNode` too. but I think that `ModelViewProjectMatrixNode` is much big... `MVP` is an academical  abbreviation for this. http://www.opengl-tutorial.org/beginners-tutorials/tutorial-3-matrices/

Reply:
We have to consider that the result is not output as Matrix but as Transformed Vertex....

Reply:
WVP... My dyslexia hit hard ... I'll fix it.

Reply:
I think I prefer the name `MVPNode`. 

@mrdoob Do you have a preference here?

Reply:
I vote to `GLPositionNode`. I think that is more simpler and say more about the output be a `vec4` since I use one `mvpMatrix * vec4( position, 1.0 )` internally.

Reply:
> I think I prefer ModelViewProjectionMatrixNode. Even if it's longer, it's more educative.

Ok.. On second thought I can make a dynamic output (`mat4` / `vec4`) for this node.